Formula/vis.rb

```diff
@@ -38,12 +38,16 @@ def install
       system "luarocks", "build", "lpeg", "--tree=#{luapath}"
     end
 
-    system "./configure", "--prefix=#{libexec}"
+    system "./configure", "--prefix=#{prefix}"
     system "make", "install"
+
+    # Rename vis & the matching manpage to avoid clashing with the system.
+    man1.install man1/"vis.1" => "vise.1"
```

Comment:
`mv` is probably a little nicer

Reply:
Aye. I can never make my mind up which looks less grim in these situations, so happy with `mv` if that's your preference.

Reply:
Are we sure there's no `--progname=` for this? It seems really weird for upstream to use a system duplicate name and not make it configurable.